packages/components/organisms/f-header/test/visual/f-header.visual.desktop.spec.js

```diff
@@ -140,4 +140,28 @@ describe('Shared - f-header component tests', () => {
         // Assert
         browser.percyScreenshot('f-header - custom nav links only', 'desktop');
     });
+
+    it('should display link in hover state', () => {
+        header = new Header();
+
+        // Act
+        header.load();
+        header.hoverOverLink('help');
+
+        // Assert
+        browser.percyScreenshot('f-header - hover state', 'desktop');
+    });
+
+    it('should display link in focus state', () => {
+        header = new Header();
+
+        // Act
+        header.load();
+        browser.keys('\uE004');
+        browser.keys('\uE004');
+        browser.keys('\uE004');
```

Comment:
Perhaps a bit of a silly question, what what are these "keys" exactly? Is it worth extracting this value into a meaningful variable?

Reply:
these reflect the tab key - I'll extract to a variable, it's not clear at all 😄